templates/taxonomy-topic.php

```diff
@@ -1,10 +1,53 @@
 <?php
 get_header();
+
+$tags = array();
+$faqs = array();
```

Comment:
Instead of pushing each post in the main loop to `$faqs`, you could use the existing `$wp_query->posts` array, e.g.:

```php
global $wp_query;
$faqs = $wp_query->posts;
```